src/PAModelTests/ChecksumTests.cs

```diff
@@ -57,44 +58,54 @@ public void ChecksumsUnique()
         // Use const fields to help for cases when checksums should be the same. 
-        const string C1 = "C5_QbctS/y+6a/vwTGdwZgmDn39Q7FtRfVrO93tXoQn/qc=";
-        const string C2 = "C5_fiqPYqGq3wwtwdolVxxT4En9IFjhYUmYoiEPT/vaDCk=";
-        const string C3 = "C5_ungWv48Bz+pBQUDeXa4iI7ADYaOWF3qctBD/YfIAFa0=";
-        const string C4 = "C5_bDw5bta1w23K4XInH0YgUbEma4Uekt897qisZUeP1xI=";
-        const string C5 = "C5_a4ayc/80/OGda4BO/1o/V0etpOqiLx1JwB5S3beHW0s=";
-        const string C6 = "C5_U5T6p9qOA98Qs/n3bCKNf9lQNHwIAJKYfZHR2rnO6eg=";
-        const string C7 = "C5_EqC8ygsfMLdppTgi6G0gq5uj01kbJFxsSPckbfr462E=";
-        const string C8 = "C5_bjQLnP+zepicpUTmu3gKLHiQHT+zNzh2hRGjBhevoB0=";
-        const string C9 = "C5_ypeBEsobvcr6wjGzmiPcTaeG7/gUfE5yuYB3ha/uSLs=";
-        const string C10 = "C5_47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=";
-        const string C11 = "C5_+44g/C5MPySMYMOb1lLzwTRymLuXe4tNWQO4UFViBgM=";
-        const string C12 = "C5_3FSyml4tICY8Z4MeBmDtBUxNBGyFnChsoX3A5mSsn2k=";
+        const string C1 = "C6_VrBVZxNCyyq6nNHBFhEi1p8/+LEgauFoOxISpTMfidA=";
+        const string C2 = "C6_bSgalCJTSBUuXr/l9syvLFQtQveX9OeUkRFO+bXaxsY=";
+        const string C3 = "C6_pV4gc7whiqiJA6qNnhtYNgnoy0AV19//Bs/JF+bwvks=";
+        const string C4 = "C6_XRxPTS3gnYmfjZL2jlOD/3STXSC8VniAOx8bfu0p3Bc=";
+        const string C5 = "C6_6mPsJ1PnWkhrz7kOmeIdrtdFtkUH+WrnfOrR/YWwB4Q=";
+        const string C6 = "C6_JpWWJaI11Wjp10J8M8WQ/ZKTOLDqyuh2XpuhrnjNIZI=";
+        const string C7 = "C6_SsPqlOsAkyDo7QmBft/8uwvbbDAEpX0rfiYqNUZdC/s=";
+        const string C8 = "C6_8ZdpKBDUV+KX/OnFZTsCWB/5mlCFI3DynX5f5H2dN+Y=";
+        const string C9 = "C6_mBjUrHTZUVjb7YgLtnZAWXjp25eh8h/uN1rgzH0qTmY=";
+        const string C10 = "C6_B0Pu6r/4VrrelJ4VYXbhTNJQfG6zem/OvUxIEweFAa4=";
+        const string C11 = "C6_rNmFqjFuTa2nSPd+9J11Dkq2CE1d3UHrdRPbakKNsz8=";
+
+        const string C12 = "C6_xinyFpAi3E1fug553zcwOQphnA26HzffwtFHFYvwBIQ=";
+        const string C13 = "C6_tLvF8n0SCv0h+OVpNzeBfjvnMsfWN2SvOttGXgkzud8=";
+        const string C14 = "C6_But9amnuGeX733SQGNPSq/oEvL0TZdsxLrhtxxaTibg=";
+        const string C15 = "C6_mF50+afGew2Ea527fnP0EhsQSunAh8GbgaBbxpNPKpM=";
+        const string C16 = "C6_tpIqQWd4zNcNsjI8xQKjvoocOT+cACVDU9oEmpw/I7Y=";
+        const string C17 = "C6_nxln5Ugv128wodPRdYyn8U9mXYzeCvPefCJwkLyof8A=";
+
+        const string C18 = "C6_HyC9hHMBR+0EPsD6yYguZ+cYv2ovwlgEfh3iygNbWHM=";
+        const string C19 = "C6_f8iCQ1KTyoVxp/xpbpwNXvjEZgIeNIaTXdzTe0q26ps=";
+        const string C20 = "C6_2YUFpLVLEYtdFvV9iLN8F6TM+cWczemMx4m0VEIpfrg=";
 
 
         [DataTestMethod]
         [DataRow(C1, @"' ab\r\ncd'")] // whitespace
         [DataRow(C1, @"'ab\ncd'")] // same
         [DataRow(C2, @"'ab\ncd  '")] // Trailing whitespace is not trimmed. 
         [DataRow(C3, "{'InvariantScript' : 'abc'}")]
-        [DataRow(C3, "[ 'abc']")] // BAD: InvariantScript is skipped
+        [DataRow(C12, "[ 'abc']")] // InvariantScript is skipped
```

Comment:
The thing to notice here is that we remove a bunch of collisions now (values used to be the same, and now they're different).